code/game/objects/items/devices/radio/headset.dm

```diff
@@ -79,6 +79,12 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	. = ..()
 	.["headset"] = TRUE
 
+/obj/item/radio/headset/MouseDrop(mob/over, src_location, over_location)
+	var/mob/M = usr
+	if((M == over) && usr.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
```

Comment:
if you're going to typecast usr as a mob in line 83, then you should be calling canUseTopic on the typecasted var in 84